plugin/pkg/scheduler/api/validation/validation.go

```diff
@@ -34,10 +34,17 @@ func ValidatePolicy(policy schedulerapi.Policy) error {
 		}
 	}
 
+	binders := 0
 	for _, extender := range policy.ExtenderConfigs {
 		if extender.Weight <= 0 {
 			validationErrors = append(validationErrors, fmt.Errorf("Priority for extender %s should have a positive weight applied to it", extender.URLPrefix))
 		}
+		if extender.BindVerb != "" {
```

Comment:
This is actually the config, not the extender implementation. The config doesn't have an equivalent method.